src/main/resources/application.yml

```diff
@@ -6,7 +6,7 @@ spring:
   application:
     name: calculate-release-dates-api
   codec:
-    max-in-memory-size: 10MB
+    max-in-memory-size: 200MB
```

Comment:
this doesn't need to be this drastic, the response in dev was a little over 10MB